lib/internal/fs/promises.js

```diff
@@ -635,7 +635,7 @@ async function writeFile(path, data, options) {
     return writeFileHandle(path, data);
 
   const fd = await open(path, flag, options.mode);
-  return writeFileHandle(fd, data).finally(fd.close);
+  return PromisePrototypeFinally(writeFileHandle(fd, data), fd.close);
```

Comment:
Hey, this needs to be added to the `} = primordials;` bit on top :]

Reply:
Solved :-)